src/drawing/DrawingAnnotations.tsx

```diff
@@ -1,30 +1,86 @@
 import * as React from 'react';
+import * as uuid from 'uuid';
 import DrawingList from './DrawingList';
-import { AnnotationDrawing } from '../@types';
+import DrawingCreator from './DrawingCreator';
+import DrawingPathGroup from './DrawingPathGroup';
+import DrawingSVG, { DrawingSVGRef } from './DrawingSVG';
+import { AnnotationDrawing, PathGroup } from '../@types';
+import { CreatorItemDrawing, CreatorStatus } from '../store';
 import './DrawingAnnotations.scss';
 
 export type Props = {
     activeAnnotationId: string | null;
     annotations: AnnotationDrawing[];
+    isCreating: boolean;
     isCurrentFileVersion: boolean;
+    location: number;
     setActiveAnnotationId: (annotationId: string | null) => void;
+    setStaged: (staged: CreatorItemDrawing | null) => void;
+    setStatus: (status: CreatorStatus) => void;
+    staged?: CreatorItemDrawing | null;
 };
 
 const DrawingAnnotations = (props: Props): JSX.Element => {
-    const { activeAnnotationId, annotations, isCurrentFileVersion, setActiveAnnotationId } = props;
+    const {
+        activeAnnotationId,
+        annotations,
+        isCreating,
+        isCurrentFileVersion,
+        location,
+        setActiveAnnotationId,
+        setStaged,
+        setStatus,
+        staged,
+    } = props;
+    const [stagedRootEl, setStagedRootEl] = React.useState<DrawingSVGRef | null>(null);
+    const stagedGroupRef = React.useRef<SVGGElement>(null);
+    const uuidRef = React.useRef<string>(uuid.v4());
 
     const handleAnnotationActive = (annotationId: string | null): void => {
         setActiveAnnotationId(annotationId);
     };
+    const handleStart = (): void => {
+        if (staged === null) {
+            setStaged(null);
+        }
+        setStatus(CreatorStatus.started);
+    };
+    const handleStop = (pathGroup: PathGroup): void => {
+        const { drawnPathGroups = [] } = staged || {};
+        const payload = {
+            drawnPathGroups: drawnPathGroups.concat(pathGroup),
+            location,
+            stashedPathGroups: [],
+        };
+
+        setStaged(payload);
+        setStatus(CreatorStatus.staged);
+    };
 
     return (
-        <DrawingList
-            activeId={activeAnnotationId}
-            annotations={annotations}
-            className="ba-DrawingAnnotations-list"
-            data-resin-iscurrent={isCurrentFileVersion}
-            onSelect={handleAnnotationActive}
-        />
+        <>
+            <DrawingList
+                activeId={activeAnnotationId}
+                annotations={annotations}
+                className="ba-DrawingAnnotations-list"
+                data-resin-iscurrent={isCurrentFileVersion}
+                onSelect={handleAnnotationActive}
+            />
+
+            {staged && (
+                <DrawingSVG ref={setStagedRootEl} className="ba-DrawingAnnotations-target">
```

Comment:
Does this need to live in its own manager/layer like region creation does?

Reply:
It doesn't need to right now. Region is "special" because it had to be layered in a certain way for discoverability. If we find we need to separate them later we can do that